src/scenes/gameOver.js

```diff
@@ -0,0 +1,62 @@
+export default class GameOver extends Phaser.Scene {
+  constructor() {
+    super({ key: "gameOver" });
+  }
+
+  init(data) {
+    this.name = data.name;
+    this.number = data.number;
+    this.next = data.next;
+  }
+
+  create() {
+    this.add.image(0, 0, "menuBackground").setOrigin(0, 0);
```

Comment:
**suggestion**: Consider adding a fade-in effect for the background image to enhance the visual transition to the Game Over screen.